dev-util/conan/Manifest

```diff
@@ -1 +1 @@
-DIST conan-1.0.4.tar.gz 580404 BLAKE2B a3099e7f5bdb882591bab031da267f3df5582975d93a840cd7bf42d7b5b57850d42a28ea33440f6570149b7db8c2ae6a395460f375fee084a9336435d42e7df6 SHA512 47fe9632b0007cdc0456004e6c9122df387869c33b58694b4a2dca9956daf977532f79c204207b21d3a4e7d90337a25dba49cf84cba98c1f9bde3170f1a185a9
```

Comment:
Removals and additions belong in separate commits. However, please don't remove the old version instantly.